vendor/bat-native-ads/src/bat/ads/internal/privacy/challenge_bypass_ristretto/signed_token.h

```diff
@@ -35,7 +35,7 @@ class SignedToken {
   bool operator==(const SignedToken& other) const;
   bool operator!=(const SignedToken& other) const;
 
-  bool has_value() const { return signed_token_.has_value(); }
+  bool has_value() const { return signed_token_ && signed_token_.has_value(); }
```

Comment:
signed_token_ and signed_token_.has_value() are the same checks

Reply:
Fixed, thanks